### ISSUE TITLE
I couldn't figure out how to use the delay? Is this integrated?

### ISSUE THREAD

**Issue by ahdsr** [bitworking/react-gsap#57]

**Comment by bitworking** [comment]
Yes, you can use any Tween property directly on the Tween component as prop: https://github.com/bitworking/react-gsap/blob/dc503e4f5241a1916ef8505fe769c0910d7d90e4/packages/playground/src/examples/Timeline.tsx#L102
